tso-server/src/test/java/org/apache/omid/tso/TestBatch.java

```diff
@@ -110,49 +110,14 @@ public void testBatchFunctionality() {
         }
 
         // Test that sending replies empties the batch
-        final boolean MASTER_INSTANCE = true;
-        final boolean SHOULD_MAKE_HEURISTIC_DECISSION = true;
-        batch.sendRepliesAndReset(replyProcessor, retryProcessor, MASTER_INSTANCE);
+        batch.sendRepliesAndReset(replyProcessor, retryProcessor);
         verify(replyProcessor, timeout(100).times(BATCH_SIZE / 2))
                 .timestampResponse(anyLong(), any(Channel.class), any(MonitoringContext.class));
         verify(replyProcessor, timeout(100).times(BATCH_SIZE / 2))
-                .commitResponse(eq(!SHOULD_MAKE_HEURISTIC_DECISSION), anyLong(), anyLong(),
-                        any(Channel.class), any(MonitoringContext.class));
+                .commitResponse(anyLong(), anyLong(), any(Channel.class), any(MonitoringContext.class));
         AssertJUnit.assertFalse("Batch shouldn't be full", batch.isFull());
```

Comment:
Might be a good idea to slowly change AssertJUnit to Assert in all classes we touch